spec/spec_helper.rb

```diff
@@ -1,9 +1,13 @@
 require 'factory_girl_rails'
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.